src/client/Microsoft.Identity.Client/Internal/Requests/Silent/SilentRequest.cs

```diff
@@ -48,6 +48,13 @@ protected override async Task<AuthenticationResult> ExecuteAsync(CancellationTok
             bool isBrokerConfigured = AuthenticationRequestParameters.AppConfig.IsBrokerEnabled &&
                                       ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth();
 
+            if (isBrokerConfigured && AuthenticationRequestParameters.PopAuthenticationConfiguration != null)
+            {

```

Comment:
Would we ever get here if someone specifies ADFS authority, enables broker, enables Pop? I don't think so, right, we have checks in builders for this?

Reply:
It will get there, but im not sure that is a problem? Im assuming WAM can handle ADFS accounts? 
Also, not sure if ADFS supports pop at the moment.
@gladjohn @bgavrilMS?

Reply:
A good test case to code up.